src/Datadog.Trace/Cache/StringBuilderCache.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Text;
+
+namespace Datadog.Trace.Cache
```

Comment:
Should this go into `Datadog.Trace.Util` for now? I think `Cache` is misleading. This class is more akin to a `StringBuilder` pool (with max instances == 1) than a cache. I would suggest renaming the class as well, but I feel less strongly about that one since it's borrowed from the runtime.

Reply:
Sure, I'll move it.